src/hooks/redis.js

```diff
@@ -0,0 +1,51 @@
+import { safeRequire } from '../utils/requireUtils';
+import { hook } from '../extender';
+import { getRandomId } from '../utils';
+import { createRedisSpan, extendRedisSpan } from '../spans/redisSpan';
+import { SpansContainer } from '../globals';
+import * as logger from '../logger';
+
+const createCallbackHandler = redisSpan => args => {
+  const ended = new Date();
+  const [error, result] = args;
```

Comment:
This function is in the `beforeHook` so how can it get `error` and `result`?
Aren't they only available in the `afterHook`?

Reply:
This is the wrapping of the callback, Redis client calls customer callback with: `callback(err, data)`